### ISSUE TITLE
Stress test: Misalignment when submitting transactions in quick succession

### ISSUE THREAD

**Issue by bstr156** [syscoin/pali-wallet#250]
### What happened?

While stress testing a new bridge dapp, I discovered what appears to be a Pali Wallet (or API) bug or limitation.  The behavior described below is not isolated to the new bridge dapp, it can be reproduced on the current dapp at bridge.syscoin.org, or likely on any dapp integrated with Pali.

I ran multiple distinct sessions (three) of a dapp all using the same wallet instance (e.g. have several tabs open of bridge.syscoin.org in the same browser). I used the dapp to submit the same type of transaction in Pali back-to-back on each session (e.g. submit multiple Mint SYSX transactions, each in quick succession, moving to each tab).  RESULT: the first dapp session appears to work fine, but the other sessions do not recognize that the transaction has been submitted at all, even though they have and they are onchain with confirmations. Of course when I attempt to re-mint (redo the transactions) through Pali that fails because a mint based on that burn proof has already been done onchain, so Pali appropriately refuses it. End result is the second and third dapp sessions are both at an impasse and cannot proceed to next steps.

What is interesting is this does not happen when submitting Metamask transactions similarly, only when using Pali Wallet.

**There is more specific information available in a GitHub issue I originally posted on the repository of this new bridge dapp.  Reach out to me in Discord and I will share the link (that dapp is not public yet)**.

### Version

1.0.28

### What platform are you using?

Linux

### What browser are you using?

Firefox

### Is this a trezor issue?

No

**Comment by bstr156** [comment]
From user osiastedian:

_"I was able to reproduce the issue as described.

I think this is a PaliWallet issue on how they handle simultaneous signing request on how it translates to dapps.
MetaMask once signed it immediate returns a transaction hash for it. Unlike Pali it still waits a little while then Transaction ID is given.

I've tried executing similar behavior on another Dapp, the old Bridge.
Video demo here:
https://drive.google.com/file/d/1ZGSbkJTKzg8F47h2LlIW5YB_bU9wE_0B/view?usp=sharing

And you'll notice PaliWallet behave the same way."_
